Content.Shared/_Goobstation/Devil/DevilBranchPrototype.cs

```diff
@@ -0,0 +1,21 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._Goobstation.Devil;
+
+[Prototype("devilBranchPrototype")]
```

Comment:
Remove prototype name and change the one instance that exists to `devilBranch` instead
```suggestion
[Prototype]
```